### DISCUSSION TITLE
Long time ago sins latest update?

### DISCUSSION THREAD

**Discussion by Goten87** [louislam/dockge#585] [General]
so just asking here i dont know whats happening but i see that the latest update was in januari thats a long time ago is this project still alive or do i need to change to something else?

**Comment by twich1226** [comment]
owner seems very busy with a different project, uptime-kuma. That project seems very active with many more users. I don't think this is abandon-ware, just lower priority.

**Reply by Samillion** [reply]
Indeed. Though hopefully this project doesn't die out, it is genuinely an awesome straightforward solution compared to alternatives. The simplicity makes it perfect.

**Comment by lukyjay** [comment]
There's 22 open pull requests. Maybe someone willing could fork this project

**Reply by twich1226** [reply]
I did it but have no idea what I am doing. If I survive pulling the pulls at least we will have a maintained repo. If Louislam ever comes back to this we can merge back.

Any help in the form of reviewing is appreciated.

**Reply by twich1226** [reply]
Change of plans, not going to start another dead project. Hopefully this ones gets forked or picked up again properly.

**Reply by Triskae** [reply]
I can fork it, and merge the pending pull requests for who want's it, and as you say, @louislam merge back when he want.

**Reply by Samillion** [reply]
> I can fork it, and merge the pending pull requests for who want's it, and as you say, @louislam merge back when he want.

That would be great. I think @cmcooper1980 might be doing that as well in https://github.com/cmcooper1980/dockge

**Reply by jaeti01** [reply]
Hello @cmcooper1980, I tried to deploy your fork as an update but I am getting errors that the frontend version doesn't match thw backend. I restarted the browser, cleared browser data, still no luck. Any hint how to solve this? Thank you for your great work.

**Reply by hbilbo** [reply]
I get similar errors as @jaeti01 since arm does not seem to be supported by that fork. I get errors that the requested images platform doesn't match the detected host platform. Just curious @cmcooper1980 is there any reason that issues and discussions are disabled on your fork? Is the goal just to have the fork merged once the dev has time?

**Reply by jaeti01** [reply]
Mine runs on arm as well.

**Reply by cmcooper1980** [reply]
> Hello @cmcooper1980, I tried to deploy your fork as an update but I am getting errors that the frontend version doesn't match thw backend. I restarted the browser, cleared browser data, still no luck. Any hint how to solve this? Thank you for your great work.

I haven't looked too closely at it as I just turned checking for updates off, however, I believe it's because the code appends the `nightly` to the backend when the image is built and a compare is done to this static site: https://dockge.kuma.pet/version in the About.vue file...a few other files need to be updated like maybe the package.json and check-version.ts files...but again, it hadn't bothered me so I haven't looked too hard.

**Reply by cmcooper1980** [reply]
> I get similar errors as @jaeti01 since arm does not seem to be supported by that fork. I get errors that the requested images platform doesn't match the detected host platform. Just curious @cmcooper1980 is there any reason that issues and discussions are disabled on your fork? Is the goal just to have the fork merged once the dev has time?

Yes, was hoping the PRs would eventually be merged to official build and so I essentially just forked for personal use picking PRs that seemed useful...that and it is my first major fork so didn't know how much time I'd dedicate to it. But I think I will open those threads up as discussions amongst others may help solve any issues and as not to clutter up these threads in regards to my specific fork.

**Reply by boced66** [reply]
Hello, 

Thank you for your work @cmcooper1980, 
I have an error when I want to save a Compose file.

![image](https://github.com/user-attachments/assets/a9bb3128-2f8c-46d5-829e-dcfce42e7207)

I have configured PUID and PGID correctly.
I don't get the error with the original repository.

**Reply by cmcooper1980** [reply]
sorry, seems the merge of the gitops stuff caused issue where existing compose stacks are recognized and are controllable, however, edits and saves to compose files error with "Stack not found". Seems only adding stacks via "Manage this stack with git" work properly. I recommend using previous build with tag `dockge:2025.01.21` until I get time to look and fix or a PR to fix it is available.

**Reply by boced66** [reply]
Thanks

**Comment by BenRoe** [comment]
Is this project still being maintained? The last commit was four months ago, with 36 open PRs.
Some great features could be merged.
It would be great to have a fork with multiple maintainers.

**Reply by lukyjay** [reply]
A lot of people are now using Komodo with Gitops for automation.
E.g. https://nickcunningh.am/blog/how-to-automate-version-updates-for-your-self-hosted-docker-containers-with-gitea-renovate-and-komodo

**Reply by cmcooper1980** [reply]
> Is this project still being maintained? The last commit was four months ago, with 36 open PRs. Some great features could be merged. It would be great to have a fork with multiple maintainers.

updated fork
